plenty_api/api.py

```diff
@@ -837,6 +839,48 @@ def plenty_api_get_contacts(self,
             refine=refine,
             additional=additional)
 
+    def plenty_api_get_property_selections(self,
+                                           refine: dict = None,
+                                           additional: list = None):
+        """
+        Get maaping of selection IDs to the actual value in all avaiable
+        languages.
+
+        Parameter:
+            refine      [dict]  -   Apply filters to the request
+                                    Example:
+                                    {'email': 'a@posteo.net', 'name': 'Thomas'}
```

Comment:
Strange example those are not actual examples for the refine attributes of this call. You pretty much only explain how a dictionary looks, which is not very helpful.